pkg/minishift/registration/registration.go

```diff
@@ -62,18 +62,18 @@ func Register(name string, r *RegisteredRegistrator) {
 	registrators[name] = r
 }
 
-func DetectRegistrator(c provision.SSHCommander) (Registrator, error) {
+func DetectRegistrator(c provision.SSHCommander) (Registrator, bool, error) {
```

Comment:
+1, that works, adding an additional boolean return value. I think this is quite idiomatic in Go.